src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -54,64 +86,131 @@ export default {
+        metadataSidebarProps: defaultMetadataSidebarProps,
+    },
     play: async ({ canvasElement }) => {
         const canvas = within(canvasElement);
-        const heading = await canvas.findByRole('heading', { name: 'Metadata' });
-        expect(heading).toBeInTheDocument();
 
-        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' });
-        expect(addTemplateButton).toBeInTheDocument();
+        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' }, { timeout: 5000 });
         await userEvent.click(addTemplateButton);
 
-        const customMetadataOption = canvas.getByRole('option', { name: 'Custom Metadata' });
-        expect(customMetadataOption).toBeInTheDocument();
+        const customMetadataOption = canvas.getByRole('option', { name: 'My Template' });
         await userEvent.click(customMetadataOption);
+    },
+};
 
+export const MetadataInstanceEditorWithCustomTemplate: StoryObj<typeof MetadataSidebarRedesign> = {
+    args: {
+        fileId: '416047501580',
+        metadataSidebarProps: defaultMetadataSidebarProps,
+    },
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' }, { timeout: 5000 });
         await userEvent.click(addTemplateButton);
-        const myTemplateOption = canvas.getByRole('option', { name: 'My Template' });
-        expect(myTemplateOption).toBeInTheDocument();
-        await userEvent.click(myTemplateOption);
 
-        const unsavedChangesModal = screen.getByText('Unsaved Changes');
-        expect(unsavedChangesModal).toBeInTheDocument();
+        const customMetadataOption = canvas.getByRole('option', { name: 'Custom Metadata' });
+        await userEvent.click(customMetadataOption);
+    },
+};
+
+export const MetadataInstanceEditorCancelChanges: StoryObj<typeof MetadataSidebarRedesign> = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const editButtons = await canvas.findAllByRole('button', { name: 'Edit' }, { timeout: 5000 });
+
+        let headlines = await canvas.findAllByRole('heading', { level: 1 });
+        expect(headlines).toHaveLength(3);
+        expect(headlines.map(heading => heading.textContent)).toEqual(
+            expect.arrayContaining(['My Template', 'Select Dropdowns', 'Custom Metadata']),
+        );
+
+        // go to edit mode - only edited template is visible
+        await userEvent.click(editButtons[0]);
+
+        headlines = await canvas.findAllByRole('heading', { level: 1 });
+        expect(headlines).toHaveLength(1);
+        expect(headlines.map(heading => heading.textContent)).toEqual(expect.arrayContaining(['My Template']));
+
+        // cancel editing - back to list view
+        const cancelButton = await canvas.findByRole('button', { name: 'Cancel' }, { timeout: 5000 });
```

Comment:
I deleted the timeout here and it seems to work locally, do we need the timeout at all anymore in any of the `findByRole`?